### PR TITLE
Remove safari hack and obsolete code

### PR DESCRIPTION
**Pull request by MKaHead**
#2236 

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: src/components/10-atoms/input-text/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 6.2.1
+
+Remove safari hack to fix and obsolete code #2345
```
Comment (markus-walther) [inline src/components/10-atoms/input-text/CHANGELOG.md]: grammar: to fix **what?**

Comment (markus-walther) [reply src/components/10-atoms/input-text/CHANGELOG.md]: spelling: **S**afari

File: src/components/10-atoms/input-text/index.js [hunk 2]
```diff
@@ -366,57 +353,26 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
...
-                  ?readonly="${readonly}"
-                  @input="${this.handleInput}"
-                  @focus="${this.handleFocus}"
-                  @blur="${this.handleBlur}"
-                  @keydown="${this.onKeyDown}"
-                  @keyup="${this.onKeyUp}"
-                />
-              `
-        }
+        ${html`<input
```
Comment (markus-walther) [inline src/components/10-atoms/input-text/index.js:356]: So, the hack was needed at some point. Starting from which (mobile) Safari version this is fixed now? We would at the very least expect some info on this in the Github issue that would be closed by this PR, as a comment.

File: src/components/10-atoms/input-text/CHANGELOG.md (hunk 1 above)
Comment (markus-walther) [reply src/components/10-atoms/input-text/CHANGELOG.md]: grammar: 'to obsolete' is not an existing verb IMHO. Suggestion: ... and **remove** obsolete code (showMessage getter).

**Review by markus-walther — CHANGES_REQUESTED** [review]
Hi Marco,

good work - surprised to see the hack causing the issues in the ticket!

Some doc work, then it's done.

Keep it up, Markus

**Comment by MKaHead** [comment]
@markus-walther fix findings and add a comment in the Github issue #2236

**Review by markus-walther — APPROVED** [review]
Marco, LGTM.

--Markus

**Pull request merged**